bdai_ros2_wrappers/bdai_ros2_wrappers/action_client.py

```diff
@@ -21,6 +21,7 @@ def __init__(
         namespace: Optional[str] = None,
         context: Optional[Context] = None,
         num_executor_threads: int = 1,
+        timeout_sec: float = 10.0,
```

Comment:
This should default to the old behavior of waiting indefinitely (-1.0 perhaps?).  There's definitely a workflow where you launch stuff and then let it hang out while you get around to launching something else.

Reply:
I've put a PR fix on the `bdai` repo as an alternative solution, which can be found [here](https://github.com/bdaiinstitute/bdai/pull/700)